examples/app-basic/src/plugin.json

```diff
@@ -2,7 +2,7 @@
   "$schema": "https://raw.githubusercontent.com/grafana/grafana/master/docs/sources/developers/plugins/plugin.schema.json",
   "type": "app",
   "name": "Basic App",
-  "id": "basic-app",
+  "id": "grafana-basic-app",
```

Comment:
Prefixing with `grafana-` to remove risk of colliding.

Reply:
Isn't the prefix of a plugin id be the org name? Maybe swap it out for `myOrg`?

`"id": "myorg-basic-app",`